### ISSUE TITLE
Add publisher/journal: Federation of European Biochemical Societies

### ISSUE THREAD

**Issue by mariaguerreiro** [datadryad/dryad-product-roadmap#3295]
**Name to appear on membership page:** 
Federation of European Biochemical Societies

**Payment plan:**
Annual membership + DPCs under Wiley

**Journals included:** 
The FEBS Journal
FEBS Letters
Molecular Oncology
FEBS Open Bio

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 
See here: https://docs.google.com/spreadsheets/d/1he0FTpJqXz8YBamuCZPHN7tB_aNYOobgnN32v1ugAwg/edit?usp=sharing

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** 
See here: https://docs.google.com/spreadsheets/d/1he0FTpJqXz8YBamuCZPHN7tB_aNYOobgnN32v1ugAwg/edit?usp=sharing

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**
hmeljak@febs.org ; ORCiD: 0000-0002-8155-8000
wright@febs.org ; ORCiD: 0009-0008-7874-2422
bogeska@febs.org ; ORCiD: 0000-0002-2064-1695
fuentes@febs.org ; ORCiD: 0000-0003-1977-4316
ruffell@febs.org; ORCID: 0000-0001-9570-4462

**Email contacts to discuss integration of journal metadata with Dryad:**
Key contact: Daniela Ruffell (ruffell@febs.org)
By journal: See here: https://docs.google.com/spreadsheets/d/1he0FTpJqXz8YBamuCZPHN7tB_aNYOobgnN32v1ugAwg/edit#gid=1888868475

cc @jleighherzog 

Development tasks:
- [x] add member name to membership page
- [x] update payment settings
- [x] update journal notification settings
- [x] add journal administrators
- [ ] start integration process (track on new ticket)

**Comment by ahamelers** [comment]
Only Daniela Ruffell already has a Dryad account, I haven't been able to add the other admins.

**Comment by jleighherzog** [comment]
Sent access confirmation to Daniela. Suggested that a reminder is sent to the Managing Editors because none have logged in yet so we're unable to set up their accounts.

**Comment by jleighherzog** [comment]
@ahamelers : Please proceed to set up journal access for these FEBS contacts. The journal titles they should have access to are listed below. 

> Duncan Wright 
> 0009-0008-7874-2422
> Journal: FEBS Letters

> Ruzhica Bogeska
> 0000-0002-2064-1695
> Journal: FEBS Press (Molecular Oncology)

**Comment by jleighherzog** [comment]
@ahamelers : Please proceed to set up journal account access for Sarah Fuentes

[Sara Fuentes Perez](https://datadryad.org/stash/user_admin/user_profile/109983)
0000-0003-1977-4316
Journal: FEBS PRESS

**Comment by jleighherzog** [comment]
@ahamelers : Please proceed to set up journal account access for [Julija Hmeljak](https://datadryad.org/stash/user_admin/user_profile/110329)

Reference technical onboarding spreadsheet for specifics re: journal access: https://docs.google.com/spreadsheets/d/1he0FTpJqXz8YBamuCZPHN7tB_aNYOobgnN32v1ugAwg/edit#gid=1888868475

**Comment by jleighherzog** [comment]
**Note**: The group email contact for **The FEBS Journal** is [febsj@febs.org](mailto:febsj@febs.org), not [TheFEBSJournal@febs.org](mailto:TheFEBSJournal@febs.org) as listed in the google spreadsheet.

Please make the necessary changes.

**Comment by ahamelers** [comment]
All done!
